eng/pipelines/common/evaluate-default-paths.yml

```diff
@@ -29,8 +27,6 @@ jobs:
     - subset: mono
       include:
       - src/libraries/System.Private.CoreLib/*
-      - src/libraries/Native/Unix/System.Globalization.Native/*
-      - src/libraries/Native/Unix/Common/*
```

Comment:
installer path partition should exclude `src/native/libs/*`?

Reply:
coreclr builds static host. Technically, it can be affected by changes in libs.native (by means of entrypoints.h, which p/invoke dispatcher relies on).

Reply:
Well, if changes under `src/libraries/Native` happened before this change, the installer subset was not tested, so I think that's fine.

Reply:
@safern, this comment is on mono subset, did you mean to point out mono or installer?

Reply:
I meant installer, I can't comment on installer since the GH diff won't let me since it wasn't changed.

Reply:
installer does not include `src/native/libs/*`.

Reply:
It does, look at my other comments. Basically installer is defined as this:

```yml
    - subset: installer
      include:
      - docs/manpages/*
      exclude:
      - eng/Version.Details.xml
      - '*.md'
      - LICENSE.TXT
      - PATENTS.TXT
      - THIRD-PARTY-NOTICES.TXT
      - docs/*
      - src/coreclr/*
      - src/mono/*
      - src/libraries/*
      - src/tests/*
      - src/native/eventpipe/*
      - eng/pipelines/coreclr/*
      - eng/pipelines/mono/*
      - eng/pipelines/libraries/*
```

Which means, include everything but what's excluded in that list + `docs/manpages/*` the reason why `docs/manpages/*` needs to be in the `include` list is because we explicitly exclude: `docs/*`. 

So that means that it will include everything under `src/native/*` except `src/native/eventpipe/*` so that is why I suggest adding and exclusion to this list to exclude `src/native/libs/*`

Reply:
Ah, gotcha. Just so you know, this code refers to libs.native stuff:
```
src/native/corehost/apphost/static/CMakeLists.txt
src/native/corehost/hostpolicy/hostpolicy_context.cpp
```
I understand that you are saying that it's fine to ignore building corehost when stuff changes in libs.